linkml/utils/helpers.py

```diff
@@ -1,3 +1,8 @@
 def remove_duplicates(lst):
     """Remove duplicate tuples from a list of tuples."""
     return [t for t in (set(tuple(i) for i in lst))]
+
+def write_to_file(file_path, data, mode="w"):
```

Comment:
seems a bit overkill to have a helper class - or is the idea this is where we abstract over ensuring everything is utf-8?

Reply:
Oh, it's not a class? I thought `linkml.utils.helpers` module was where we put in all our helper methods, that we can use within our core generators, validators, transformers, etc.?

Reply:
sorry, I meant to say helper method

Reply:
I would expect that such helper method would be easily overlooked - as easy as it is to forget adding `encoding="utf-8"` like below :-) . Unfortunately the filesystem encoding cannot be changed. So discovering this in tests is difficult unless you want to activate testing on windows in github actions.